### DISCUSSION TITLE
Solid support

### DISCUSSION THREAD

**Discussion by timredd** [barvian/number-flow#39] [Ideas]
Awesome work, `NumberFlow`'s very polished and I appreciate the framework-agnostic core. I see support for Vue is planned, is there any interest for [SolidJS](https://docs.solidjs.com/quick-start) support?. If so, I've got a local branch that I can open a PR for to get feedback on. Thanks!

**Comment by barvian** [comment]
Hey, thanks for the kind words! I'm a big admirer of Solid but I haven't built much with it so I'm not sure I'm the right person to tackle this atm. I'm definitely not ruling it out, though. I know @Blankeos has pushed [solid-number-flow](https://github.com/Blankeos/solid-number-flow) so maybe that's worth looking into. In the future I want to make it easier for other people to spin up framework wrappers for NumberFlow. Converting this to a discussion for now so it can be voted on 👋
